website/docs/tutorials/podtato-head.md

```diff
@@ -0,0 +1,124 @@
+---
+id: podtato-head
+title: Let's start with Podtato-head
+sidebar_label: Podtato-head
+---
+
+---
+
+![podtato-head](../assets/tutorials/podtato-head/podtato-head.png)
+
+In this tutorial, you will inject a pod-delete fault into a sample microservices application called [podtato-head](https://github.com/cncf/podtato-head) and verify if the service continues to be available during the chaos duration.
+
+## Prerequisites
+
+- Kubernetes 1.18 or later (minimum 2 vCPUs, 8GB RAM, 10GB disk space)
+
+- A Persistent volume of 20GB
+
+- [Helm3](https://v3.helm.sh/) or [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+## Install ChaosCenter
+
+1. Follow the [Getting-started](../getting-started/installation.md) to install ChaosCenter
+
+2. Access to ChaosCenter Dashboard
+
+![chaoscenter-dashboard](../assets/tutorials/podtato-head/chaoscenter-dashboard.png)
+
+## Set up Environment
+1. Add a new environment
+- Environment Name: `local`
+- Environment Type: `Production`
+
+![local-environment](../assets/tutorials/podtato-head/local-environment.png)
+
+## Enable Chaos Infrastructure
+
+1. Configure a new chaos infrastructure
+- Name: `local`
+- Chaos Components Installation: `Cluster-wide access`
+- Installation Location (Namespace): `litmus`
+- Service Account Name: `litmus`
+
+2. Deploy the new chaos infrastructure
+
+```bash
+kubectl apply -f local-litmus-chaos-enable.yml
+```
+
+3. Wait until the status shows `CONNECTED`.
+
+![connected](../assets/tutorials/podtato-head/connected.png)
+
+## Set up Resilience Probe
+
+1. Select HTTP Probe as the probe type
+
+2. Configure properties & probe details
+- Name: `check-podtato-main-access-probe`
+- Timeout: `10s`
+- Interval: `1s`
+- Attempt: `1`
+- URL: `http://podtato-main.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local:9000`
+- Method: `GET`
+- Criteria: `==`
+- Response Code: `200`
+
+![setup-probe](../assets/tutorials/podtato-head/setup-probe.png)
+
+## Run Chaos Experiment with Podtato-head
+1. For Kubernetes v1.24 or later, bind a ClusterRole to the `argo-chaos` service account 
```

Comment:
why are we binding argo-chaos serviceAccount to admin cluster-role?

Reply:
Starting with Kubernetes v1.24, service accounts no longer receive tokens automatically upon creation. To provide the necessary permissions to the `argo-chaos` service account, we initially bound it to the `admin` cluster role. However, since this cluster role grants excessively broad permissions, it would be more appropriate to assign a role specific to the `litmus` namespace instead. 

Additionally, please note that with the recent update to the tutorial, this step is now unnecessary.